src/in_memory.rs

```diff
@@ -83,12 +83,18 @@ impl TermLike for InMemoryTerm {
 
     fn write_line(&self, s: &str) -> std::io::Result<()> {
         let mut state = self.state.lock().unwrap();
+
+        // Don't try to handle writing lines with additional newlines embedded in them - it's not
+        // worth the extra code for something that indicatif doesn't even do. May revisit in future.
+        assert!(
```

Comment:
Maybe this should be a `debug_assert!()`?

Reply:
Good point, I'll change it